packages/tendermint-rpc/src/jsonrpc.ts

```diff
@@ -20,8 +20,7 @@ function randomId(): number {
 }
 
 /** Creates a JSON-RPC request with random ID */
-// eslint-disable-next-line @typescript-eslint/ban-types
-export function createJsonRpcRequest(method: string, params?: {}): JsonRpcRequest {
+export function createJsonRpcRequest(method: string, params?: params?: Record<string, any>): JsonRpcRequest {
```

Comment:
Fix my own messup. Sorry
```suggestion
export function createJsonRpcRequest(method: string, params?: Record<string, any>): JsonRpcRequest {
```